src/DependencyInjection/Configuration.php

```diff
@@ -111,7 +112,9 @@ public function getConfigTreeBuilder()
                                 ->thenInvalid('Configuration value of "global_search.empty_boxes" must be one of show, fade or hide.')
                             ->end()
                         ->end()
+                        // NEXT_MAJOR: Remove this option.
                         ->booleanNode('case_sensitive')
+                            ->setDeprecated('The "%node%" option is deprecated since sonata-project/admin-bundle 3.x.')
```

Comment:
I'm not sure, but IMO we should use "configuration" instead of "option" in this sentence.

Reply:
I copy pasted the message from another deprecated option.

And in the doc it was
```
To search case-insensitive use the following option:

.. code-block:: yaml
    # config/packages/sonata_admin.yaml
    sonata_admin:
        global_search:
            case_sensitive: false
```

So seems like we're calling this an "option".